dali/pipeline/operators/fused/normalize_permute.cc

```diff
@@ -70,6 +70,9 @@ Normalization takes input image and produces output using formula
 ..
 
 output = (input - mean) / std
+
+*Deprecated*: Use CropMirrorNormalize_ (without crop and mirror arguments) instead,
```

Comment:
Maybe we should crash inside the `NormalizePermute` constructor with the `Deprecated` message so use will know that he should move to `CropMirrorNormalize`.

Reply:
```suggestion
*Deprecated*: Use :meth:`nvidia.dali.ops.CropMirrorNormalize` (without crop and mirror arguments) instead,
```

Reply:
Maybe only having a warning for this version, and make it crash on the next one?

Reply:
Ok. Let's go with warning first.

Reply:
Done